<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>0</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 17pt &quot;Segoe UI&quot;;
color:rgb(0, 0, o);
</string>
   </property>
   <property name="text">
    <string>Araç Kiralama Sistemi</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>571</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="styleSheet">
       <string notr="true">font: 700 italic 12pt &quot;Segoe UI&quot;;
</string>
      </property>
      <property name="text">
       <string>Müşteri Bilgileri</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="styleSheet">
       <string notr="true">font: 700 italic 12pt &quot;Segoe UI&quot;;
</string>
      </property>
      <property name="text">
       <string>Araç Bilgileri</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="styleSheet">
       <string notr="true">font: 700 italic 12pt &quot;Segoe UI&quot;;
</string>
      </property>
      <property name="text">
       <string>Kiralama Bilgileri</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_4">
      <property name="styleSheet">
       <string notr="true">font: 700 italic 12pt &quot;Segoe UI&quot;;
</string>
      </property>
      <property name="text">
       <string>Kiralama Listesi</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
